linera-chain/src/block.rs

```diff
@@ -192,3 +206,362 @@ pub enum ConversionError {
     #[error("Expected a `TimeoutCertificate` value")]
     Timeout,
 }
+
+/// Block defines the atomic unit of growth of the Linera chain.
+/// As part of the block body, contains all the incoming messages
```

Comment:
I think it's common to add a blank line (i.e. new paragraph in markdown; also below).

```suggestion
///
/// As part of the block body, contains all the incoming messages
```